packages/prisma-client/prisma/migrations/20230501151941_asset-to-file/migration.ts

```diff
@@ -0,0 +1,43 @@
+import { PrismaClient } from "./client";
+
+export default () => {
+  const client = new PrismaClient({
+    // Uncomment to see the queries in console as the migration runs
+    // log: ["query", "info", "warn", "error"],
+  });
+  return client.$transaction(
+    async (prisma) => {
+      const result = await prisma.$executeRaw`
+        INSERT INTO "File"
+        (
+          "name",
+          "format",
+          "size",
+          "description",
+          "createdAt",
+          "meta",
+          "status"
+        )
+        SELECT
+          "name",
+          "format",
+          "size",
+          "description",
+          "createdAt",
+          "meta",
+          "status"
+        FROM "Asset"
+        GROUP BY
```

Comment:
why? do we have duplicates?

Reply:
Yes, project cloning reuse same files in storage and duplicate only entries in db.